### DISCUSSION TITLE
AdvancedUrlRewriter: Unknow duplicate code

### DISCUSSION THREAD

**Discussion by thienvc** [dnnsoftware/Dnn.Platform#4425] [Q&A]
I found duplicate call function IgnoreRequest(result, fullUrl, ignoreRegex, request)

https://github.com/dnnsoftware/Dnn.Platform/blob/4689315a681dc87b04944ac9a0c15cabf63168fa/DNN%20Platform/Library/Entities/Urls/AdvancedUrlRewriter.cs#L2037
https://github.com/dnnsoftware/Dnn.Platform/blob/4689315a681dc87b04944ac9a0c15cabf63168fa/DNN%20Platform/Library/Entities/Urls/AdvancedUrlRewriter.cs#L2116

Can someone tell me why?

**Comment by mitchelsellers** [comment]
Looking at the code, it appears that first the raw request is compared to see if we need to ignore it.  If not ignored UrlRewriting is done etc to get it to the proper URL>

The result of that operation is then re-evaluated as the rewrite might change something?  

Not sure "why" though.
